packages/wallet-adapter-react/src/useWallet.tsx

```diff
@@ -49,7 +50,7 @@ export interface WalletContextState {
   ): Promise<any>;
   signMessage(message: SignMessagePayload): Promise<SignMessageResponse | null>;
   signMessageAndVerify(message: SignMessagePayload): Promise<boolean>;
-
+  submitTransaction(transaction: InputGenerateTransactionData): Promise<any>;
```

Comment:
Okay... we need to make this better.

`any` doesn't cut it anymore.  We should at least make it return a hash string or an object with a hash string so it can be extended accordingly.